### ISSUE TITLE
Fb BOTS ARE BACK!!!!

### ISSUE THREAD

**Issue by idkRage** [davidmann4/ogar-feeder-bot#596]
Fb Bots are back chither.com
i find something go to agar.io and tipe in browser console dfpInitialized=facebookLogin

**Comment by MasonBurdette** [comment]
Yes, there is a algorithm to convert an access token to proxyMobileData.

**Comment by MinersMC** [comment]
@MasonBurdette 
var token = ""
  var bytes = [8, 1, 18, token.length + 25, 1, 8, 10, 82, token.length + 20, 1, 8, 2, 18, 13, 8, 5, 18, 5, 50, 46, 49, 46, 50, 24, 0, 32, 0, 26, token.length, 1];

  for (var i = 0; i < token.length; i++) bytes.push(token.charCodeAt(i));

  setTimeout(function() {
  window.core.proxyMobileData(new Uint8Array(bytes));
  }, 1000);

**Comment by MasonBurdette** [comment]
@MinersMC I wouldn't give that out publicly...

**Comment by MinersMC** [comment]
@MasonBurdette but i just did lol

**Comment by MinersMC** [comment]
just deleted the massage lol

**Comment by ImHITMan** [comment]
I know this topic is for agario, But is it possible to make the Ogar-feeder bot made by Davidmann4 and make it work on Mgar.io?

**Comment by MasonBurdette** [comment]
@ImHITMan easily

**Comment by Nick8fr32** [comment]
i already did lol XD @lmangani

**Comment by ImHITMan** [comment]
@MasonBurdette What would you have to do? I've been told that You will have to change OPCodes and protocol And some other stuff,

**Comment by ImHITMan** [comment]
@MasonBurdette Would it be possible for you to do that for me? I'd really appreciate it a lot.

**Comment by op-client** [comment]
give me algorithm to convert an access token to proxyMobileData
